js/gulp/minify-task.js

```diff
@@ -74,15 +74,16 @@ const uglifyTask = ((cache, commonConfig) => memoizeTask(cache, function uglifyJ
                         .replace(/\s/, `_`)
                         .replace(/\.\/node_modules\//, ``)
             }),
-            new UglifyJSPlugin({
+            new TerserPlugin({
                 sourceMap: true,
-                uglifyOptions: {
-                    ecma: uglifyLanguageNames[target],
+                terserOptions: {
+                    ecma: terserLanguageNames[target],
                     compress: { unsafe: true },
                     output: { comments: false, beautify: false },
-                    mangle: { eval: true, safari10: true, // <-- Works around a Safari 10 bug: // https://github.com/mishoo/UglifyJS2/issues/1753
+                    mangle: { eval: true,
                         properties: { reserved, keep_quoted: true }
-                    }
+                    },
+                    safari10: true // <-- works around safari10 bugs, see the "safari10" option here: https://github.com/terser-js/terser#minify-options
```

Comment:
@trxcllnt do you have access to safari so you can validate if this still works?

Reply:
Yes, I'm on it. What's a good example to try?

Reply:
Hmm, I don't think it's possible to install an old version but I would be surprised if the patch that once worked doesn't work anymore.